docs/deprecated.md

```diff
@@ -14,17 +14,15 @@ keywords: "docker, documentation, about, technology, deprecate"
      will be rejected.
 -->
 
-# Deprecated Engine Features
-
 The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
 ### Asynchronous `service create` and `service update`
 
-**Deprecated In Release: v17.05.0**
+## Deprecated In Release: v17.05.0
```

Comment:
I'm not sure this is correct, actually, now the version in which a feature is deprecated has a bigger heading than the feature that _is_ deprecated, causing the hierarchy to be wrong. In this case, for example, the feature that is deprecated is "Asynchronous `service create` and `service update`", and it was marked deprecated in release 17.05, and the default will change in release 17.09.

So either the "deprecated in release .." needs to be changed to a H4, or kept as "bold".

(We should probably have a look in general if we can present this in a more readable way)

Reply:
Sorry for the delay. My semester started a couple of weeks ago. Ok, I can change all the "deprecated in release .." to H4 and we can see how that looks.

Reply:
There is also "Target For Removal In Release" in the doc. I'm guess you want these to be H4 as well.